.circleci/config.yml

```diff
@@ -0,0 +1,51 @@
+version: 2
+jobs:
+  test:
+    docker:
+      - image: artsy/hokusai:0.4.0
+    steps:
+      - add_ssh_keys
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Test
+          command: hokusai test
+  push:
+    docker:
+      - image: artsy/hokusai:0.4.0
+    steps:
+      - add_ssh_keys
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Push
+          command: hokusai registry push --tag $CIRCLE_SHA1 --force --overwrite
+  deploy:
+    docker:
+      - image: artsy/hokusai:0.4.0
+    steps:
+      - add_ssh_keys
+      - checkout
+      - run:
+          name: Configure
+          command: hokusai configure --kubectl-version 1.6.3 --s3-bucket artsy-citadel --s3-key k8s/config --platform linux
+      - run:
+          name: Deploy
+          command: hokusai staging deploy $CIRCLE_SHA1
+workflows:
+  version: 2
+  default:
+    jobs:
+      - test
+      - push:
```

Comment:
I'm not 100% sure, but Iit looks like we should replace the `only: hokusai` with `only: master`?

ie- we want to deploy from master.

Maybe we don't need `push:` job?

Reply:
Ah I need to replace `hokusai` with `master` again.  We need the push job to build and push the image